pom.xml

```diff
@@ -202,14 +226,126 @@
                 <id>copy-dependencies</id>
                 <phase>package</phase>
                 <goals>
-                  <goal>copy-dependencies</goal>
+                  <goal>copy</goal>
+                  <!--<goal>copy-dependencies</goal>-->
```

Comment:
I'm guessing this was changed from copy-dependencies to just copy to reduce the number of jars included in libext.